test/react-web/client/graphql/mutations.test.tsx

```diff
@@ -394,4 +394,218 @@ describe('mutations', () => {
     renderer.create(<ApolloProvider client={client}><Container id={'123'} /></ApolloProvider>);
   });
 
+  it('will run `updateQueries` for a previously mounted component', () => new Promise((resolve, reject) => {
```

Comment:
This test is really long and keeps track of a lot of stuff, so it would be useful to have a short comment here that summarizes what it does.